### ISSUE TITLE
Upload Modal Triggering Library Refresh

### ISSUE THREAD

**Issue by jauyong** [GoogleForCreators/web-stories-wp#1201]
## Bug Description

After closing the upload modal, the media library is refreshing. As we know this has some performance / loading issues. 

## Expected Behaviour

Experience should be smooth and not sure if this should trigger a refresh in the library.

## Steps to Reproduce

open and close the upload modal.

## Screenshots

![upload modal](https://user-images.githubusercontent.com/5874851/79257910-56667b00-7e58-11ea-91c3-a830680da6f5.gif)

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

**Comment by spacedmonkey** [comment]
In the media dialog, lots of different functions can be done by the user. For example, alt text can be edited, images deleted and new files to can be uploaded. So it was seen as a good idea to refresh the library. However, in earlier versions, the loading message didn't show and the refresh was a little bit more graceful. Transmission between the two states is a smoother way the current. But we are waiting on #662 #439 

Thoughts @merapi ?

**Comment by diegovar** [comment]
@samitron7 has asked us to not show the loading message if the wait time is short, so this might be a non-issue once we do that.

**Comment by samitron7** [comment]
Correct - users should never see the loading message for any loading state in the editor (not just media) if loading is quick. @jhtin is setting a duration time for this of 500ms. I'm not sure what that perfect time is. @pbakaus if you have a suggestion.

**Comment by pbakaus** [comment]
> Correct - users should never see the loading message for any loading state in the editor (not just media) if loading is quick. @jhtin is setting a duration time for this of 500ms. I'm not sure what that perfect time is. @pbakaus if you have a suggestion.

Seems OK. Let's try it out and see how it feels.

**Comment by diegovar** [comment]
Closing as we've implemented a delay for the loading pill.

**Comment by swissspidy** [comment]
Reopening as the media library is not actually refreshing right now after uploading media through the WordPress media library (via the upload button).

/cc @diegovar - can you confirm that?

**Comment by diegovar** [comment]
Closing as there's another ticket for this issue.
